picard/util/__init__.py

```diff
@@ -459,4 +458,20 @@ def convert_to_string(obj):
         return str(obj)
 
 
+def htmlescape(string):
+    return cgi.escape(string)
+
+
+def json_load(data):
+    """
+    Appropriately convert and returns a python dictionary from
+    json string.
```

Comment:
According to https://tools.ietf.org/html/rfc7159#section-7 a JSON string is just a sequence of characters enclosed in quotation marks, the input expected should be described similarly to how the `json.loads` function does it.